src/debug/Debug.js

```diff
@@ -0,0 +1,112 @@
+import dat from 'dat.gui/build/dat.gui';
```

Comment:
Yeah I should have left a comment. Because we then need to process dat's files through babel, and we use incompatible babel configs with dat.gui :-/ It's possible I didn't find the way out though. If you have better, I'd be glad to hear it!